docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3"
+networks:
+  db-data-net:
+    driver: "bridge"
+services:
+  pythonapp:
+    image: app:1.0
```

Comment:
The image name `app:1.0` should be verified to ensure it exists in your Docker registry. If it's a local build, ensure the Dockerfile is correctly set up to build this image.